openedx/core/lib/block_structure/tests/helpers.py

```diff
@@ -127,7 +130,11 @@ def create(cls, children_map):
         """
         modulestore = MockModulestore()
         modulestore.set_blocks({
-            block_key: MockXBlock(block_key, children=children, modulestore=modulestore)
+            block_key_factory(block_key): MockXBlock(
```

Comment:
**Note to reviewers:** The introduction of `block_key_factory` was needed in the tests since the `data_usage_key` in `BlockStructureModel` is of type `UsageKeyField` and requires a valid `UsageKey` object.  Prior to this, the tests in this layer didn't need to use real `OpaqueKey` types (and just used integer values).